### PR TITLE
Add skip-filter flag for summarizer

### PR DESCRIPTION
**Pull request by chases2**
Adds a flag that allows you to turn off regex filtering (if it's already being done upstream by the tabulator.

### CODE REVIEW

File: pkg/summarizer/summary.go [hunk 1]
```diff
@@ -750,39 +749,37 @@ func firstFilled(values ...int32) int {
...
+func filterGrid(baseOptions string, rows []*statepb.Row, recent int, useRegexFilter bool) ([]*statepb.Row, error) {
 
 	rows = recentRows(rows, recent)
 
 	rows = filterMethods(rows)
 
-	for _, include := range vals[includeFilter] {
-		if rows, err = includeRows(rows, include); err != nil {
-			return nil, fmt.Errorf("bad %s=%s: %v", includeFilter, include, err)
+	if useRegexFilter {
```
Comment (michelle192837) [inline pkg/summarizer/summary.go:764]: nit: Maybe `if !useRegexFilter { return rows, nil }` to limit what's inside the conditional?

**Review by michelle192837 — APPROVED** [review]
/hold

Unhold at will

File: pkg/summarizer/summary.go (hunk 1 above)
Comment (chases2) [reply pkg/summarizer/summary.go]: Good idea; done

**Pull request merged**